dayforce_client/__init__.py

```diff
@@ -1 +1,6 @@
-from dayforce_client.client import Dayforce, DayforceSFTP  # noqa: F401
+from dayforce_client.client import (
+    Dayforce,
+    DayforceSFTP,
+    ImportError,
+    ImportPending,
+)  # noqa: F401
```

Comment:
@bobzoller since we need to ignore more than a single line now, let's disable flake8 qa a different way. Let's add it to the flake8 config [here](https://github.com/goodeggs/dayforce-client/blob/master/tox.ini#L9) as suggested in the [flake8 docs]((http://flake8.pycqa.org/en/3.1.1/user/ignoring-errors.html#ignoring-entire-files)). Something like:

```python
[flake8]
ignore =
  E501
count = True
statistics = True
exclude = .venv/**,.venvs/**,.tox,dayforce_client/__init__.py
```